easybuild/framework/easyconfig/templates.py

```diff
@@ -73,6 +73,7 @@
 TEMPLATE_NAMES_EASYBLOCK_RUN_STEP = [
     ('installdir', "Installation directory"),
     ('builddir', "Build directory"),
+    ('module_name_full', "Full module name"),
```

Comment:
wouldn't `full_module_name` make more sense?

Reply:
depends how you look at it :) either way is fine to me.